python/GafferUI/CompoundEditor.py

```diff
@@ -1278,3 +1376,96 @@ def _reprDict( d ) :
 		"'%s' : %s" % ( k, IECore.repr(v) ) for k,v in d.items()
 	] )
 
+
+# A little color swatch that can be inserted into a _TabbedContainer to
+# represent the driven/driving state of the current editor
+from GafferUI.Frame import Frame as _Frame
+class _DrivenEditorSwatch( _Frame ) :
+
+	__drivenEditorColors = [
+		imath.Color3f( 0.27, 0.45, 0.21 ),
+		imath.Color3f( 0.71, 0.43, 0.47 ),
+		imath.Color3f( 0.85, 0.80, 0.48 ),
+		imath.Color3f( 0.62, 0.79, 0.93 )
+	]
+	__drivenEditorColorsLastUsed = 0
+	__drivenEditorColorMapping = {}
+
+	def __init__( self ) :
+
+		_Frame.__init__( self, borderWidth = 0, borderStyle = GafferUI.Frame.BorderStyle.None )
+		self._qtWidget().setFixedSize( 6, 6 )
+
+	def getToolTip( self ) :
+
+		editor = self.__getNodeSetEditor()
+		if editor is None :
+			return ""
+
+		masterEditor = editor.drivingEditor()
+		drivenEditors = editor.drivenNodeSets( recurse = True ).keys()
+
+		toolTipElements = []
+
+		if masterEditor is not None :
+			toolTipElements.append( "Following _%s_" % masterEditor.getTitle() )
+
+		if drivenEditors :
+			toolTipElements.append( "Folllowed by:")
```

Comment:
`toolTipElements.append( "Followed by:")`

Reply:
Fixed in https://github.com/GafferHQ/gaffer/commit/aca4e28c8ebb8804581443491d1e9f96db3798b5